src/lib/Card/Card.tsx

```diff
@@ -0,0 +1,66 @@
+import clsx from "clsx";
+import React from "react";
+import { CardImageBorderRadius, CardImageSize, CardProps } from "./Card.types";
+import { Button } from "../Button";
+import { ButtonSize, ButtonType, ButtonVariant } from "../Button/Button.types";
+import "./Card.scss";
+
+export const Card: React.FC<CardProps> = ({
```

Comment:
Mainly for the storybook docs but it might be nice to have some small documentation on this component just explaining its purpose, similar to the other components in the repo 😄